card/mod/05_standard/spec/chunk/include_spec.rb

```diff
@@ -16,7 +16,7 @@
       expect(render_content("{{# -->}}")).to eq('<!-- # --&gt; -->')
     end
 
-    it "handles empty inclusions" do
+    it "handles empty nests" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.